modules/permutiveRtdProvider.js

```diff
@@ -70,11 +70,12 @@ export function setBidderRtb (auctionDetails, customModuleConfig) {
   const moduleConfig = getModuleConfig(customModuleConfig)
   const acBidders = deepAccess(moduleConfig, 'params.acBidders')
   const maxSegs = deepAccess(moduleConfig, 'params.maxSegs')
+  const transformationConfigs = deepAccess(moduleConfig, 'params.transformations') || []
   const segmentData = getSegments(maxSegs)
 
   acBidders.forEach(function (bidder) {
     const currConfig = bidderConfig[bidder] || {}
-    const nextConfig = mergeOrtbConfig(currConfig, segmentData)
+    const nextConfig = mergeOrtbConfig(currConfig, segmentData.ac, transformationConfigs) // ORTB2 uses the `ac` segment IDs
```

Comment:
Does this mean we will be applying the IAB transform for other SSPs asides from Index?

Reply:
We have agreed to check with Hearst

Reply:
I think it's okay as long as we run it by Hearst first and they agree.
If that isn't the case, I guess we can add a `bidders` property to the transformation config and use it to only attach the transformed data for the appropriate bidders.